.gitignore

```diff
@@ -8,7 +8,6 @@ build/
 
 # TypeScript build artifacts
 *.tsbuildinfo
-*.d.ts
```

Comment:
This file does not need to be ignored. All this does is define global types that can be referred to from anywhere in the project.